packages/evm/test/registry/guardians-registry-staking.js

```diff
@@ -459,7 +459,7 @@ contract('GuardiansRegistry', ([_, guardian, anotherGuardian, governor]) => {
       })
```

Comment:
I didn't notice any tests for ANY_ENTITY activators, which may be another sign that we may want to remove it :)